src/zac/sass/components/_result.scss

```
@@ -0,0 +1,33 @@
+.result {
+  margin-bottom: 1rem;
+
+  &__title {
+    margin: 0 0 3px 0;
+    font-size: 1rem;
+  }
+
+  &__value {
+    margin: 0;
+  }
+}
+
+.subresult {
+  &__title {
+    font-weight: bold;
+    display: inline-block;
+    margin: 0 0.4rem 0 0;
+    &:after {
+      content: ':'
+    }
+  }
+
+  &__value {
+    margin: 0;
+    display: inline-block;
+  }
+
+  &__row {
+    margin-left: 2rem;
+    margin-bottom: 2rem;
+  }
+}
```

```suggestion
@import "../lib/bem";

.result {
  margin-bottom: 1rem;

  @include element("title") {
    margin: 0 0 3px 0;
    font-size: 1rem;
  }

  @include element("value") {
    margin: 0;
  }
}

.subresult {
  @include element("title") {
    font-weight: bold;
    display: inline-block;
    margin: 0 0.4rem 0 0;
    &:after {
      content: ':'
    }
  }

  @include element("value") {
    margin: 0;
    display: inline-block;
  }

  @include element("row") {
    margin-left: 2rem;
    margin-bottom: 2rem;
  }
}
```

this mixin ensures that the resulting CSS selector is `.block .block__element` rather than `.block__element` - the latter makes it possible to write markup with elements without a parent block. The strict CSS enforces proper markup/class names structure.